modules/calypsoify/style-gutenberg.scss

```diff
@@ -154,4 +154,10 @@ a {
 		margin-top: inherit;
 		height: 100%;
 	}
-}
\ No newline at end of file
+}
+
+@media (min-width: 782px) {
+	body.is-iframed.is-fullscreen-mode .edit-post-layout__content {
```

Comment:
Could we remove the `.is-iframed` selector to make it applicable for non iframed Calypsoify'd requests too?